server/sonar-server/src/main/java/org/sonar/server/es/NewIndex.java

```diff
@@ -97,7 +100,7 @@ public NewIndexType createLongField(String fieldName) {
     }
 
     public NewIndexType createDynamicNestedField(String fieldName) {
-      return setProperty(fieldName, ImmutableMap.of("type", "nested", "dynamic", "true"));
+      return setProperty(fieldName, ImmutableMap.of("type", "nested", "dynamic", "true", "include_in_parent", "true"));
```

Comment:
Something I used during my tests, I remove it. It changes how ES index my nested type. It includes nested documents in the parent document. For example, I could do a query referencing coverageBlocks.uuid directly, without using a nested query.
More information on [elastic search website](http://www.elastic.co/guide/en/elasticsearch/reference/1.x/mapping-nested-type.html)